src/apps/lwaftr/lwutil.lua

```diff
@@ -49,3 +51,16 @@ function keys(t)
    return result
 end
 
+
+-- Local bindings for constants that are used in the hot path of the
```

Comment:
OK I know you are getting conflicting feedback here, but what about switching to the strategy in lwaftr.lua?  i.e. operating on pointers rather than absolute offsets.